code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -264,3 +302,75 @@
 	if(!cultie)
 		return FALSE
 	return cultie.total_sacrifices >= target_amount
+
+/datum/objective/ascend
+	name = "ascend"
+
+/datum/objective/ascend/update_explanation_text()
+	. = ..()
+	explanation_text = "Applease the Gods and ascend."
+
+/datum/objective/ascend/check_completion()
+	if(!owner)
+		return FALSE
+	var/datum/antagonist/heretic/cultie = owner.has_antag_datum(/datum/antagonist/heretic)
+	if(!cultie)
+		return FALSE
+	return cultie.ascended
+
+/datum/objective/minicult
+	name = "mini cult"
+
+/datum/objective/minicult/update_explanation_text()
+	. = ..()
+	target_amount = rand(2,5)
+	explanation_text = "Raise and maintain a cult of [target_amount] people."
+
+/datum/objective/minicult/check_completion()
+	if(!owner)
+		return FALSE
+	var/datum/antagonist/heretic/cultie = owner.has_antag_datum(/datum/antagonist/heretic)
+	if(!cultie)
+		return FALSE
+	return cultie.get_cur_followers()>=target_amount
+
+//////////////
+// Minicult //
+//////////////
+
+/datum/antagonist/heretic/proc/get_max_followers()
```

Comment:
Also you should just have this on a var and add to it when you unlock a knowledge that adds followers_incremenet